src/components/Header/HeaderAction/HeaderAction.jsx

```diff
@@ -112,7 +112,7 @@ const HeaderAction = ({ item, index }) => {
   // Otherwise render a simple menu button with no wrapper div
   return (
     <HeaderGlobalAction
-      className={`${carbonPrefix}--header-action-btn`}
+      className={[`${carbonPrefix}--header-action-btn`, item.className].filter(i => i).join(' ')}
```

Comment:
what's happening with this filter method?

Reply:
basically `classnames()` 😎  i'll replace it with `classnames()`